piquasso/fermionic/fock/state.py

```diff
@@ -106,6 +106,17 @@ def validate(self):
     def density_matrix(self):
         return self._connector.np.outer(self._state_vector, self._state_vector.conj())
 
+    @property
+    def fock_probabilities_map(self) -> dict:
```

Comment:
Optional: I think it is better to put this directly under `fock_probabilities_map`.